apps/verticalhome/js/app.js

```diff
@@ -327,7 +403,35 @@
         // A hashchange event means that the home button was pressed.
         // The system app changes the hash of the homescreen iframe when it
         // receives a home button press.
+        case 'timeformatchange':
```

Comment:
You inserted this new case right after a comment that is supposed to go with the next case.  Please fix that.  It would also be good to add a comment explaining where the timeformatchagne event comes from, since it does not appear anywhere else in this app directory.